### ISSUE TITLE
Problems with circuit current sensors

### ISSUE THREAD

**Issue by lars-viklund** [fondberg/easee_hass#203]
### Question

Since yesterday all my sensors for circuit current (circuitTotalPhaseConductorCurrentL1, etc.) report 0 even while charging. Is anyone else having the same issue? I know that Easee has been having some issues, but their status page claims that everything should be back to normal. Nothing related to the easee integrations in the logs.

### What version of the integration are you using?

v0.9.44

**Comment by olalid** [comment]
Yes, I am seeing the same thing here, but I can not say that I look at those sensors regularly in any case. I think the would make most sense if you have more than one charger on a circuit which I do not have.
The current sensor for the charger seems to still work normally (although a bit slow sometimes recently).
I.e. sensor.xxxx_current -> state inCurrentT3, T4, T5.

In any case this would be an issue with the cloud rather than the Easee component, but I will forward the info to Easee.

**Comment by joelsvensson** [comment]
I have the same issue

I'm using it to set the Dynamic Circuit Current so it caused me some issues when it's not updating.
Maybe I should use the T3,4,5 instead. Can anyone point me to a good reference describing/mapping those to L1/L2/L3 as I could not find any?

However it's not zero, just very small numbers and by the looks of it, it's not just a decimal change.

Output from API request (filtered) - https://developer.easee.cloud/reference/get_api-chargers-id-state
``` 
{
    "outputPhase": 30,
    "dynamicCircuitCurrentP1": 12.0,
    "dynamicCircuitCurrentP2": 15.0,
    "dynamicCircuitCurrentP3": 15.0,
    "latestPulse": "2022-08-29T19:19:56Z",
    "chargerFirmware": 300,
    "voltage": 236.475997924805,
    "inCurrentT2": 0.9850000143051147,
    "inCurrentT3": 12.079999923706055,
    "inCurrentT4": 12.067999839782715,
    "inCurrentT5": 12.04699993133545,
    "outputCurrent": 12.0,
    "inVoltageT2T3": 236.475997924805,
    "inVoltageT2T4": 233.593994140625,
    "inVoltageT2T5": 229.436004638672,
    "circuitTotalPhaseConductorCurrentL1": 0.00700000021606684,
    "circuitTotalPhaseConductorCurrentL2": 0.00899999961256981,
    "circuitTotalPhaseConductorCurrentL3": 0.00899999961256981,
}
```

**Comment by olalid** [comment]
T3 for instance means the third screw terminal on your Easee charger, so it depends on what is connected to that terminal, which in turn depends on what type of electric grid you have where you live.
Look at page 12 in https://easee.com/wp-content/uploads/2021/02/EN_Easee_HomeCharge_Guide.pdf
If you live in Sweden you will most likely have it connected according to "TN/TT 3-phase" picture.
Note that this is is the actual current used, not the setting for the dynamic circuit current limit.

That setting can instead be read from the dynamic circuit limit sensor and its attributes:
<img width="412" alt="Screenshot 2022-08-29 at 22 36 09" src="https://user-images.githubusercontent.com/34105674/187294476-72557b06-d8e8-449b-9183-f7a544060543.png">

**Comment by joelsvensson** [comment]
Thanks for a proper explanation 👍
